fiducial_slam/src/map.cpp

```diff
@@ -511,7 +516,7 @@ void Map::publishTf()
 void Map::update()
 {
     ros::Time now = ros::Time::now();
-    if (havePose && tfPublishInterval != 0.0 &&
+    if (publishPoseTf && havePose && tfPublishInterval != 0.0 &&
```

Comment:
Is the tf publish interval check redundant here?

Reply:
Almost but not exactly. The tfPublishInterval is used when you want fiducial_slam to re-publish the TF when a fiducial isn't detected. For example, you may want TF's to be published only when a fiducial is detected so you'd set publishPoseTf=True and tfPublishInterval=0.0.

Reply:
@rohbotics Need anything else to keep the ball rolling?